serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -8,6 +8,37 @@ toc::[]
 
 If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
 
+[id="update-api-group_{context}"]
+== Updating the API group
```

Comment:
I'd make this section a separate procedure module. It sounds like you need to perform these steps after you upgrade, so I'd move it after that section.